web/views.py

```
@@ -6928,6 +6931,184 @@ def all_study_groups(request):
+
+
+@login_required
+def work_submission_detail(request: HttpRequest, submission_id: int) -> HttpResponse:
+    """View to see the details of a specific work submission and provide feedback."""
+    submission = get_object_or_404(WorkSubmission, pk=submission_id)
+    # Determine user role
+    is_student = request.user == submission.student
+    is_reviewer = hasattr(request.user, "profile") and getattr(request.user.profile, "is_teacher", False)
+    # Check if the user has access
+    if not (is_student or is_reviewer):
+        raise PermissionDenied()  # Use default message or define in exception class
```

Ruff (0.8.2)

Unnecessary parentheses on raised exception

Remove unnecessary parentheses

(RSE102)